contrib/post-receive

```diff
@@ -12,9 +12,16 @@ echo "\n==> Activating virtualenv…\n"
 . /opt/jarbas.venv/bin/activate
 
 echo "\n==> Installing NodeJS packages…\n"
+if [ -d node_modules/ ]; then
+    rm -rf node_modules/
+fi
+if [ -d elm-stuff/ ]; then
+    rm -rf elm-stuff/
+fi
 yarn install
 
 echo "\n==> Installing Python packages…\n"
+pip install -U pip
 pip install -r requirements.txt
```

Comment:
That's an idea, but… `requirements.txt` _should_ have precise versions, not `wahtever>=1.0.0`.

That said, I don't think it's _necessary_, but I won't oppose this change.

What about you: _that said_, can I add the `-u` or maybe I have just convinced you otherwise?